### DISCUSSION TITLE
Notation CLI UX improvements

### DISCUSSION THREAD

**Discussion by yizha1** [notaryproject/notation#299] [Ideas]
This discussion will be used to collect Notation CLI UX improvements. It will contain two parts:
- All the existing issues related to CLI UX
- Innovative ideas for CLI UX design or improvements

The outcome will be a few new Issues, PRs, and plan for release.

I suggest using label `UX` for all the UX related issues

@FeynmanZhou @shizhMSFT @dtzar @iamsamirzon

**Comment by yizha1** [comment]
Here are four issues addressing notation sign, verify, CLI error message and CLI help doc
- #91, several items mentioned in this issue, suggest creating separate issues with `UX` label
- #228 
- #128 
- #224

**Comment by iamsamirzon** [comment]
Good discussion thread @yizha1 . I am adding one more for consideration

- https://github.com/notaryproject/notation/issues/300

**Comment by yizha1** [comment]
Two new issues created to improve notation sign and verify UX 

- #303 
- #304
